nmrfx-core/src/main/java/org/nmrfx/chemistry/io/MMcifWriter.java

```diff
@@ -75,18 +75,18 @@ static String getMainDirectory() {
     static void makeWeightMap() throws IOException {
```

Comment:
This function is unused, the only other possible reference to it is this function (because it takes a parameter called weightMap) which is also unused: https://github.com/nanalysis/nmrfx/blob/a6fc42ab4eb3036172576cd6c7eb9d0b9e672131/nmrfx-core/src/main/java/org/nmrfx/chemistry/Residue.java#L836

Can these two functions be removed

Reply:
They are not currently being used so could be removed.  I won't be surprised if we go back to working on read/write of MMcif files that we'll need to find them.  But doing that isn't currently a priority.